src/napari_spatialdata/_interactive.py

```diff
@@ -213,8 +213,10 @@ def _add_label(self, key: str) -> None:
         selected_cs = self.coordinate_system_widget._system
         affine = _get_transform(self._sdata.labels[key], selected_cs)
 
+        rgb_labels, rgb = _transform_to_rgb(element=self._sdata.labels[key])
```

Comment:
I find this function name a bit confusing in this context as we don't want to transform the labels to RGB. Not a major blocker, but is there a more clear name we can use?

Also, I assume we aren't using the `rgb` flag here. If that is the case, I suggest (minor nitpick):
```suggestion
        rgb_labels, _= _transform_to_rgb(element=self._sdata.labels[key])
```